js/main.js

```diff
@@ -641,8 +641,13 @@ function htmlEncode(s) {
           .replace(/>/g, '&gt;');
 }
 
+// Convert hex string and opacity to an rgba string
+function hexToRGBA(hex, opacity) {
+	return 'rgba(' + (hex = hex.replace('#', '')).match(new RegExp('(.{' + hex.length/3 + '})', 'g')).map(function(l) { return parseInt(hex.length%2 ? l+l : l, 16) }).concat(opacity||1).join(',') + ')';
```

Comment:
Could you please change indentation here to 2 spaces?

Reply:
Hi. Yes I will. I'm also addressing the failed check. Cheers.

Reply:
The problems reported by codacy seem to be false positives